.eslintrc.js

```diff
@@ -1,17 +1,21 @@
 module.exports = {
+  root: true,
   env: {
     browser: true,
     es2021: true,
   },
   extends: [
+    "eslint:recommended",
     "airbnb-base",
-    "plugin:cypress/recommended",
```

Comment:
Why remove this?

Reply:
I've moved it into the eslintrc in the cypress directory